### ISSUE TITLE
Bitwarden Beta Manifest 3 Extension is not compatible with desktop app

### ISSUE THREAD

**Issue by FadeMind** [bitwarden/clients#9134]
### Steps To Reproduce

1. Install and setup Bitwarden Desktop.
2. Enable Browser Integration in Bitwarden Desktop
3. Enable Unlock with biometrics in Bitwarden Chrome Beta Extension Settings

### Expected Result

Windows Hello Prompt

### Actual Result

Error - Extension can't see Desktop app. 

### Screenshots or Videos

![20240510-1715359255](https://github.com/bitwarden/clients/assets/9846948/4e8b5bb5-3a08-43e7-b64c-9fccbf4ddb88)
![20240510-1715359231](https://github.com/bitwarden/clients/assets/9846948/a90fd36c-d5f1-422d-98fc-e8dbe2468fd0)

### Additional Context

In short:
Stable version works.
Beta have error and  don't see connection between Extension AND Desktop app. 

### Operating System

Windows

### Operating System Version

Windows 11 Pro 23H2 22631.3527 X64

### Web Browser

Chrome

### Browser Version

124.0.6367.156

### Build Version

2024.4.5775.24354 beta - 4877f08e

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Neonwarden** [comment]
Hi there,

I am unable to reproduce this issue, but it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by FadeMind** [comment]
@Neonwarden Chrome  Bitwarden Beta Ext:
Video (click for download and watch localy): https://github.com/bitwarden/clients/assets/9846948/fef2839f-7449-458c-bc78-340d628cf0dc

Chrome Bitwarden Stable settings:
![20240511-1715415014](https://github.com/bitwarden/clients/assets/9846948/92ffbb71-e25b-47ed-a01a-7c37160b8c39)

**Comment by differsthecat** [comment]
Thank you for the report!

Unfortunately, biometrics are not supported in the beta at this time. 
They will be supported in manifest v3 when we start the official release in the coming days and weeks.
